File: HealthSysHub.Web.DBConfiguration/Models/HospitalContent.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace HealthSysHub.Web.DBConfiguration.Models
{
    [Table("HospitalContent")]
    public class HospitalContent
    {
        [Key]
        public Guid HospitalContentId { get; set; }
        public Guid? HospitalId { get; set; }
        public string? Description { get; set; }
        public Guid? CreatedBy { get; set; }
        public DateTimeOffset CreatedOn { get; set; }
        public Guid? ModifiedBy { get; set; }
        public DateTimeOffset? ModifiedOn { get; set; }
        public bool IsActive { get; set; }
    }
}

File: HealthSysHub.Web.Pharmacy.UI/src/app/models/hospital-content.ts
export interface HospitalContent {
  hospitalContentId: string;
  hospitalId?: string | null;
  description?: string | null;
  createdBy?: string | null;
  createdOn: Date | string;
  modifiedBy?: string | null;
  modifiedOn?: Date | string | null;
  isActive: boolean;
}
